client/containers/DashboardSettings/CommunitySettings.tsx

```diff
@@ -222,6 +228,65 @@ const CommunitySettings = () => {
 						setDescription(evt.target.value.substring(0, 280).replace(/\n/g, ' '));
 					}}
 				/>
+				<InputField
+					label={
+						<span>
+							Journal Citation
+							<Tooltip
+								content={
+									<span>
+										When filled out, this field will be used as the journal
+										title for
+										<br />
+										issue collections in citations, PDF exports, and Crossref
+										deposits.
+										<br />
+										By default, the Community Title is used.
+									</span>
```

Comment:
Sorry for being sloppy when I wrote the copy! Just a few things to capitalize in the text that I should have seen the first time. Also, I'll defer to @3mcd on this, but my sense is we shouldn't be using line breaks, as they're unpredictable at different monitor sizes. Instead, we should make sure the tooltips have a good width set, or something? But I could be totally wrong.

```When filled out, this field will be used as the Journal Title for Issue Collections in citations, PDF exports, and Crossref deposits.```